recipes-graphics/gtk+/gtk+3_%.bbappend

```diff
@@ -1,5 +1,6 @@
 DEPENDS:append:imxgpu2d = " virtual/egl"
 
-PACKAGECONFIG:remove:imxgpu2d = " \
-    ${@bb.utils.contains("DISTRO_FEATURES", "wayland", "x11", "", d)} \
-"
+WAYLAND = "${@bb.utils.contains('DISTRO_FEATURES', 'x11', '', 'wayland', d)}"
+WAYLANDONLY = "${@bb.utils.contains('DISTRO_FEATURES', 'wayland', '${WAYLAND}', '', d)}"
+
+PACKAGECONFIG:remove:imxgpu2d = " ${WAYLANDONLY}"
```

Comment:
You are right first like should put x11 instead of waylaid on rhs will send a patch soon

Reply:
It seems a more complicated way of doing the same thing? The original would remove x11 if it wasn't there. The new just doesn't remove it if it's not there?

Reply:
x11 and wayland are not mutually exclusive distro features sadly, which means originally it will remove x11 support if wayland was in distro features. it did not check for x11.

Reply:
Some packages will not build if both wayland and x11 support is configured. This is certainly true recently for Vivante GPU drivers as x11 support is dropped, so I fear that this change will cause a build break.

Reply:
It could fallback to xwayland.